rotkehlchen/db/filtering.py

```diff
@@ -1622,6 +1622,37 @@ def make(
         return filter_query
 
 
+class CounterpartyAssetMappingsFilterQuery(DBFilterQuery):
```

Comment:
This class needs a docstring. Would need to explain what's the counterparty and what's the counterparty symbol and how they are differenttoo.